data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```
@@ -414,6 +414,7 @@
       [ "coat_lab", 20 ],
       [ "helmet_army", 40 ],
       [ "helmet_army_outdated", 30 ],
+      [ "helmet_soviet_tanker", 30 ],
```

```suggestion
      [ "helmet_soviet_tanker", 3 ],
```
Cut down to 10% of chance of a WW2 US Helmet